presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcClient.java

```diff
@@ -83,9 +83,9 @@ public void testMetadata()
         assertEquals(table.get().getTableName(), "NUMBERS");
         assertEquals(table.get().getSchemaTableName(), schemaTableName);
         assertEquals(jdbcClient.getColumns(session, table.orElse(null)), ImmutableList.of(
-                new JdbcColumnHandle("TEXT", JDBC_VARCHAR, VARCHAR, true),
-                new JdbcColumnHandle("TEXT_SHORT", JDBC_VARCHAR, createVarcharType(32), true),
-                new JdbcColumnHandle("VALUE", JDBC_BIGINT, BIGINT, true)));
+                new JdbcColumnHandle("TEXT", JDBC_VARCHAR, VARCHAR, true, Optional.empty()),
```

Comment:
I recommend doing something like https://github.com/prestosql/presto/pull/1897 so that we don't need to modify all the places.